app/components/money-amount-input/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { WalletCurrency } from "@app/graphql/generated"
+import { useDisplayCurrency } from "@app/hooks/use-display-currency"
+import { DisplayCurrency, MoneyAmount, WalletOrDisplayCurrency } from "@app/types/amounts"
+import { StyleProp, TextStyle } from "react-native"
+import { FakeCurrencyInput } from "react-native-currency-input"
+
+type MoneyAmountPaymentInputProps = {
+  moneyAmount: MoneyAmount<WalletOrDisplayCurrency>
+  setAmount?: (moneyAmount: MoneyAmount<WalletOrDisplayCurrency>) => void
+  style?: StyleProp<TextStyle>
+  editable?: boolean
+}
+
+export const MoneyAmountInput = ({
+  moneyAmount,
+  setAmount,
+  style,
+  editable,
+  ...props
+}: MoneyAmountPaymentInputProps) => {
```

Comment:
eventually there should be a dependency on language also. 

in European language, currency is after the number.

running this here gives a good explanation: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

```
> "123.456,79 €"
> "￥123,457"
> "1,23,000"
```

I don't think we should do this for now. just giving this for context.

Reply:
actually right now we put sats the European way, and dollars/other fiat American way. not sure why this has emerged naturally. probably me been confused by the 2 standards.

Reply:
I think since sats is not a symbol it made sense to put it after the currency